evap/student/templates/student_vote.html

```diff
@@ -200,6 +221,18 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 bottomCard: document.querySelector("#bottom_text_results_publish_confirmation_card"),
             };
 
+            // Ensure that selected questionnaire language is saved and loaded
```

Comment:
Maybe not for this PR, but it would probably be worthwhile to test this with selenium

Reply:
Yeah, probably a good follow-up issue.